data/json/items/melee/spears_and_polearms.json

```diff
@@ -601,6 +601,27 @@
     "qualities": [ [ "COOK", 1 ] ],
     "flags": [ "SPEAR", "REACH_ATTACK", "DURABLE_MELEE" ]
   },
+  {
+    "id": "pollaxe",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": { "str": "pollaxe" },
+    "description": "This is a polearm with an axe head, a small hammer on the back, and a short dagger-like spike on top.  It's less heavy then most polearms and has a smaller head, making it slightly quicker.",
```

Comment:
```suggestion
    "description": "This is a polearm with an axe head, a small hammer on the back, and a short dagger-like spike on top.  It's less heavy than most polearms and has a smaller head, making it slightly quicker.",
```

Reply:
thank you so much i didn't notice it before!